rules/windows/powershell/powershell_cmdline_reversed_strings.yml

```diff
@@ -0,0 +1,48 @@
+title: Suspicious PowerShell Cmdline
+id: b6b49cd1-34d6-4ead-b1bf-176e9edba9a4
+description: Detects the PowerShell command lines with reversed strings
+status: experimental
+references:
+    - https://2019.offzone.moscow/ru/report/hunting-for-powershell-abuses/
+    - https://speakerdeck.com/heirhabarov/hunting-for-powershell-abuse?slide=66
+tags:
+    - attack.defense_evasion
+    - attack.t1027
```

Comment:
Hello Vasiliy!
Thank you for your contribution.
Please add this tags:
 - attack.execution
 - attack.t1059.001

Reply:
Hi! Done.